ocs_ci/ocs/platform_nodes.py

```diff
@@ -1716,9 +1757,10 @@ def attach_volume(self, volume, node):
         )
 
     def wait_for_volume_attach(self, volume):
-        raise NotImplementedError(
-            "Wait for volume attach functionality is not implemented"
-        )
+        # TODO: To check whether the given volume has been attached for the given timeout
+        # TODO: replace time.sleep with the attach check
+        time.sleep(300)
```

Comment:
Please create an issue to track the change

Reply:
Now the TODOs have been implemented!